_includes/masthead.html

```diff
@@ -15,7 +15,7 @@
   </p>
   </div>
 
-  {% endif %}
```

Comment:
We should revert this change.

Reply:
Oh. I think I explained poorly. 
We should have an `endif` for each of the if statements ( line 3 and line 11) making both the statements separate. 



In internal docs, they are nested, so they will have both `endif` at the end of the 2nd if statement (that is line 18 and line 19 in internal docs).

Reply:
I committed.